src/main/java/frc/robot/motor/MotorController.java

```diff
@@ -171,6 +174,10 @@ public default void setPID(Gains gains) {
      */
     public boolean getReverseLimit();
 
+    public default CANSparkMax asCANSparkMax() {
+        return null;
```

Comment:
This should return null by default, as should only return something if it is a Spark Motor Controller.